src/screens/DomainScreen/elements/Header/index.js

```diff
@@ -177,7 +177,7 @@ const styles = StyleSheet.create({
     position: 'absolute',
     width: 10,
     height: 10,
-    backgroundColor: COLORS.white,
+    backgroundColor: COLORS.almostBlack,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.white` to `COLORS.almostBlack` for the `backgroundColor` property in the unnamed style might affect readability if the text color within this component is not adjusted accordingly. Please ensure that the text color contrasts well with this new background color for better accessibility and readability.
```diff
-    backgroundColor: COLORS.white,
+    backgroundColor: COLORS.almostBlack,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->